app/coffee/Features/User/UserController.coffee

```diff
@@ -15,13 +15,35 @@ settings = require "settings-sharelatex"
 
 module.exports = UserController =
 
+	# TODO: deprecated, remove
```

Comment:
Any reason to not just remove now? I think the risk of people on the account page, but not hit 'delete' yet is small.